### PR TITLE
Added tests for user admin

### PR DESCRIPTION
**Pull request by bartczak-pa**

### CODE REVIEW

File: users/tests/user_admin_test.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+def test_ordering(model_admin: admin.ModelAdmin) -> None:
+    assert model_admin.ordering == ("email",)
+
+
+def test_fieldsets(model_admin: admin.ModelAdmin) -> None:
+    assert model_admin.fieldsets == ()
+
+
+def test_add_fieldsets(model_admin: admin.ModelAdmin) -> None:
```
Comment (jacoor) [inline users/tests/user_admin_test.py]: Good test.

File: users/tests/user_admin_test.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+def test_admin_registered(model_admin: admin.ModelAdmin) -> None:
+    assert isinstance(model_admin, CustomUserAdmin)
+
+
+def test_list_display(model_admin: admin.ModelAdmin) -> None:
+    assert model_admin.list_display == DjangoUserAdmin.list_display[1:]
+
+
+def test_ordering(model_admin: admin.ModelAdmin) -> None:
```
Comment (jacoor) [inline users/tests/user_admin_test.py]: This is a good exercise but not a needed test. You are testing what is provided and tested by Django (ordering)

File: users/tests/user_admin_test.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+def model_admin() -> admin.ModelAdmin:
+    return get_registered_model_admin(User)
+
+
+def test_admin_registered(model_admin: admin.ModelAdmin) -> None:
+    assert isinstance(model_admin, CustomUserAdmin)
+
+
+def test_list_display(model_admin: admin.ModelAdmin) -> None:
+    assert model_admin.list_display == DjangoUserAdmin.list_display[1:]
```
Comment (jacoor) [inline users/tests/user_admin_test.py]: This is a good exercise but not a needed test. You are testing what is provided and tested by Django. Admin Panel that is.

File: users/tests/user_admin_test.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    return model_admin_class
+
+
+@pytest.fixture()
+def model_admin() -> admin.ModelAdmin:
+    return get_registered_model_admin(User)
+
+
+def test_admin_registered(model_admin: admin.ModelAdmin) -> None:
+    assert isinstance(model_admin, CustomUserAdmin)
```
Comment (jacoor) [inline users/tests/user_admin_test.py]: Good one. Making sure it is our model.

**Review by jacoor — APPROVED** [review]
Approving, but please review comments. Some of those tests are good exercises but not needed in long run.

File: pyproject.toml [hunk 5]
```diff
@@ -45,4 +46,9 @@ line-ending = "auto"
     "S101", # asserts allowed in tests...
     "ARG", # Unused function args -> fixtures nevertheless are functionally relevant...
     "FBT", # Don't care about booleans as positional arguments in tests, e.g. via @pytest.mark.parametrize()
-]
\ No newline at end of file
+]
+
+[tool.pytest.ini_options]
```
Comment (jacoor) [inline pyproject.toml]: Please move it to pytest.ini - people are more familiar with that file.

**Review by jacoor — CHANGES_REQUESTED** [review]

**Comment by jacoor** [comment]
Actually, a change request

**Comment by jacoor** [comment]
@bartczak-pa any movement here?

File: users/tests/user_admin_test.py (hunk 2 above)
Comment (bartczak-pa) [reply users/tests/user_admin_test.py]: This will be deleted with oncoming commit.

File: pyproject.toml (hunk 5 above)
Comment (jacoor) [reply pyproject.toml]: Please add pytest.ini .

**Review by jacoor — APPROVED** [review]

**Comment by jacoor** [comment]
@bartczak-pa please merge

**Pull request merged**